pkg/metrics/probemetrics/probemetrics.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package probemetrics
+
+import (
+	"sync"
+
+	"github.com/cilium/tetragon/pkg/metrics"
+	"github.com/cilium/tetragon/pkg/metrics/consts"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type Key struct {
+	id     uint32
+	attach string
+}
+
+type Stat struct {
+	policy string
+	missed float64
+}
+
+var (
+	MissedProbes = metrics.NewBPFCounter(prometheus.NewDesc(
```

Comment:
To have this metric documented in the metrics reference, we need to create a separate dummy collector that reports this metrics with example labels. See https://github.com/cilium/tetragon/commit/861489a02bac3971ce07374b3d64e3db6b2554f4 for an example, and [consts.go](https://github.com/cilium/tetragon/blob/main/pkg/metrics/consts/consts.go) for example labels.

This is not ideal tbh. I'm working on a better metrics library for Tetragon use cases, so hopefully soon it will be more straightforward.

Reply:
sure, will add that